ui/src/app/app-routing.module.ts

```diff
@@ -4,14 +4,14 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
 
-import {CohortBuilderComponent} from 'app/views/cohort-builder/component';
-import {CohortBuilderPlaceholderComponent} from 'app/views/cohort-builder-placeholder/component';
+import {CohortBuilderComponent} from 'app/views/cohort-builder/search/cohort-builder/cohort-builder.component';
 import {CohortEditComponent} from 'app/views/cohort-edit/component';
 import {HomePageComponent} from 'app/views/home-page/component';
 import {LoginComponent} from 'app/views/login/component';
 import {WorkspaceComponent} from 'app/views/workspace/component';
 import {WorkspaceEditComponent} from 'app/views/workspace-edit/component';
 import {SelectRepositoryComponent} from 'app/views/select-repository/component';
+import { CohortReviewComponent } from 'app/views/cohort-builder/review/cohort-review/cohort-review.component';
```

Comment:
That's my whole point. You are saying we should change something but no decision has ever been made on this. These are things we all need to talk about and decide. And in this case, no spaces breaks the default CLI behavior.